research/context/preselection/preselect_wikipedia_comments.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tqdm import tqdm
+import pandas as pd
+import tensorflow as tf
+
+# Connect to Google's BigQuery
+from google.cloud import bigquery
+client = bigquery.Client(project="wikidetox-viz")
```

Comment:
Make the project a flag; our goal is to make the interface boundary (for re-use) at the shape of data, not at the ownership of projects.

Reply:
Done, I followed that also for the table_name and other important variables (e.g., min text len).